viewer/packages/tools/src/Measurement/MeasurementTool.ts

```diff
@@ -395,7 +395,7 @@ export class MeasurementTool extends Cognite3DViewerToolBase {
         intersection.point
       );
       this._viewer.domElement.addEventListener('pointermove', this._handlePointerMove);
-      window.addEventListener('keydown', this._handleMeasurementCancel);
```

Comment:
Not really related to this change, just curious: Do we always call `_handleMeasurementCancel` when a key is pressed? I.e. is the measurement always cancelled when a key is pressed?

Reply:
`_handleMeasurementCancel` is called for all key is pressed when measurement tool is active, but in the bind function I check for only `Esc` key and cancel active measurement